src/test/java/org/folio/rest/impl/POLineTest.java

```diff
@@ -0,0 +1,222 @@
+package org.folio.rest.impl;
+
+import com.jayway.restassured.RestAssured;
+import com.jayway.restassured.http.ContentType;
+import com.jayway.restassured.response.Header;
+import com.jayway.restassured.response.Response;
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import org.apache.commons.io.IOUtils;
+import org.folio.rest.RestVerticle;
+import org.folio.rest.persist.PostgresClient;
+import org.folio.rest.tools.PomReader;
+import org.folio.rest.tools.client.test.HttpClientMock2;
+import org.json.JSONObject;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.InputStream;
+
+import static com.jayway.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.*;
+
+@RunWith(VertxUnitRunner.class)
+public class POLineTest {
+  private Vertx vertx;
+  private Async async;
+  private final Logger logger = LoggerFactory.getLogger("okapi");
+  private final int port = Integer.parseInt(System.getProperty("port", "8081"));
+
+  private final String TENANT_NAME = "diku";
+  private final Header TENANT_HEADER = new Header("X-Okapi-Tenant", TENANT_NAME);
+
+  private String moduleName;      // "mod_orders_storage";
+  private String moduleVersion;   // "1.0.0"
+  private String moduleId;        // "mod-orders-storage-1.0.0"
+
+
+  @Before
+  public void before(TestContext context) {
+    logger.info("--- mod-orders-storage-test: START ");
+    vertx = Vertx.vertx();
+
+    moduleName = PomReader.INSTANCE.getModuleName();
+    moduleVersion = PomReader.INSTANCE.getVersion();
+
+    moduleId = String.format("%s-%s", moduleName, moduleVersion);
+
+    // RMB returns a 'normalized' name, with underscores
+    moduleId = moduleId.replaceAll("_", "-");
+
+    try {
+      // Run this test in embedded postgres mode
+      // IMPORTANT: Later we will initialize the schema by calling the tenant interface.
+      PostgresClient.setIsEmbedded(true);
+      PostgresClient.getInstance(vertx).startEmbeddedPostgres();
+      PostgresClient.getInstance(vertx).dropCreateDatabase(TENANT_NAME + "_" + PomReader.INSTANCE.getModuleName());
+
+    } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
Please use logger

Reply:
Changes made to use logger in both tests